### PR TITLE
Add content on follow up tasks and allow updating fields on investigation task

### PR DESCRIPTION
**Pull request by senseibara**

### CODE REVIEW

File: niger/tasks.js [hunk 1]
```diff
@@ -11,6 +11,32 @@ const descriptiveContactLabel = (contact, report) => {
...
+  content.g_details.region = Utils.getField(report, 'g_alterations.alt_region') || Utils.getField(report, 'g_details.region');
+  content.g_details.district = Utils.getField(report, 'g_alterations.alt_district') || Utils.getField(report, 'g_details.district');
+  content.g_details.commune = Utils.getField(report, 'g_alterations.alt_commune') || Utils.getField(report, 'g_details.commune');
+  content.g_details.quartier = Utils.getField(report, 'g_alterations.alt_quartier') || Utils.getField(report, 'g_details.quartier');
+  content.g_symptoms.symptoms = Utils.getField(report, 'g_alterations.alt_symptoms') || Utils.getField(report, 'g_symptoms.symptoms');
+  content.g_symptoms.other_symptoms = Utils.getField(report, 'g_alterations.alt_other_symptoms') || Utils.getField(report, 'g_symptoms.other_symptoms');
+  content.g_symptoms.call_outcome = Utils.getField(report, 'g_alterations.alt_call_outcome') || Utils.getField(report, 'g_symptoms.call_outcome');
+  return content;
+};
+const lightContent = (content, contact, report) => {
```
Comment (kennsippell) [inline niger/tasks.js:35]: consider just inlining this short function directly into the task?

File: niger/tasks.js [hunk 2]
```diff
@@ -11,6 +11,32 @@ const descriptiveContactLabel = (contact, report) => {
   return label;
 };
 
+const completeEditableContent = (content, contact, report) => {
+  content.g_details = {};
+  content.g_symptoms = {};
+  content.g_details.case_number = Utils.getField(report, 'g_alterations.alt_case_number') || Utils.getField(report, 'g_details.case_number');
```
Comment (kennsippell) [inline niger/tasks.js:17]: What do you think of something like this? I think making this all a bit more concise would really improve readability.

```
const getFields = (report, ...fields) => {
  return fields.find(field => Utils.getField(report, field));
};
```

`content.g_details.case_number = getFields(report, 'g_alterations.alt_case_number', 'g_details.case_number');`

**Comment by kennsippell** [comment]
Seems like adding a test like this one would be really valuable for the investigation form change.  https://github.com/medic/config-msf/blob/master/niger/test/tasks/riposte-followup.spec.js#L82. 

Those tests can help manage the stringiness... Would you be able to get that working?

File: niger/tasks.js (hunk 2 above)
Comment (senseibara) [reply niger/tasks.js:17]: yes that's a great suggestion, will do

File: niger/tasks.js (hunk 1 above)
Comment (senseibara) [reply niger/tasks.js]: done

**Comment by senseibara** [comment]
will work on adding the test but merging this first to not hold deployment

**Pull request merged**